Composer/packages/client/src/components/CreationFlow/FileSelector.tsx

```diff
@@ -349,7 +349,7 @@ export const FileSelector: React.FC<FileSelectorProps> = (props) => {
     },
     {
       key: 'Edit',
-      name: '',
+      name: 'Edit',
```

Comment:
We should wrap this in a `formatMessage`.
```suggestion
      name: formatMessage('Edit'),
```

Reply:
Thanks, fixed